---
title: 'ZEAMAP2'
description: 'ZEAMAP is a database for Zea genomics and maize genetic improvements by integrating the maize multidimensional data including genomics, transcriptomics, genetic variations, genetic mappings and evolutionary selection sites.'
pubDate: 'Mar 13 2024'
heroImage:
    url: '/zeamap.svg'
    alt: 'GitHub wallpaper'
platform: WEB
stack: ['Python', 'Django', 'Vue', 'Rust', 'Docker', 'Nginx']
website: https://db.cngb.org/zeamap
# github: https://github.com/kirontoo/astro-theme-cody
order: 2
---

## Summary

As one of the most extensively cultivated crops, maize (Zea mays L.) has been extensively studied by researchers and breeders for over a century. With advances in high-throughput detection of various omics data, a wealth of multi-dimensional and multi-omics information has been accumulated for maize and its wild relative, teosinte. Integration of this information has the potential to accelerate genetic research and generate improvements in maize agronomic traits. To this end, we constructed ZEAMAP, a comprehensive database incorporating multiple reference genomes, annotations, comparative genomics, transcriptomes, open chromatin regions, chromatin interactions, high-quality genetic variants, phenotypes, metabolomics, genetic maps, genetic mapping loci, population structures, and populational DNA methylation signals within maize inbred lines. ZEAMAP is user friendly, with the ability to interactively integrate, visualize, and cross-reference multiple different omics datasets.

## Contact

ZEAMAP was designed and developed by the MaizeLab of Huazhong Agricultural University and China National GeneBank DataBase . If you need to report errors, provide features or provide feedback, please contact us.

E-mails: CNGBdb@cngb.org ; wjwei@webmail.hzau.edu.cn
